pkg/agent/types/kontext.go

```diff
@@ -0,0 +1,30 @@
+package types
+
+import "go.etcd.io/bbolt"
+
+// Kontext ("Kolide context") holds references to data and structs which are used throughout
+// launcher code and are typically valid for the lifetime of the application instance.
+type Kontext struct {
```

Comment:
I'll roll with if the team wants, but I am slightly hesitant to name it `kontext`. I think it servers a very different purpose than `context` and I think the name collision will be more frustrating than joyful.

(I would support `kontext` as a wrapper over context.Context`)

Reply:
I'm not attached to it -- `runtime` feels in the right area, but perhaps is too generic or could be confused with other things? I'm open to ideas... a name that helps indicate that this struct is a collection of common objects.

Reply:
* `launchinator` ... `lchntr := launchinator.New(...)`
* `kolidergeist` ... `kgst := kolidergeist.New(...)`
* `kaunchpad` ... `kpd := kaunhcpad.New(...)`
* `kore` ... `kr := kore.New(...)`
* `knexus` ... `kxs := knexus.New(...)`

generated with some AI help =)

Reply:
I like `knapsack` -- like it's our inventory of useful services. I think in general bag/container metaphors make the most sense to me for this usage. (So maybe `kontainer`?)

I think we're at risk of overloading `agent`/`settings/`config` because those are already in use elsewhere in our codebase and mean different things, so I probably wouldn't go with something in that direction. I also really like `kontext` but agree with seph that the conflation with `context` will probably be a little confusing.

Reply:
Some sillier bag/container-based options:
* `capsule` / `kapsule` (bonus encapsulation reference)
* `locker` (looks a little similar to launcher, which is fun)
* `canteen` and `cauldron` are fun but I think they're too unreadable if you sub the c for a k, unfortunately

Reply:
+1 `knapsack`

@RebeccaMahany 's "inventory of useful services" description is pretty good.

Reply:
`knapsack` it is!